docs/Apps.md

```diff
@@ -21,7 +21,7 @@ The **web-based PoS app** allows users with brick and mortar stores to readily *
 
 Adding new products is easy. The app has a **shopping cart feature**, **tips**, **product inventory**, **custom payment options** and more.
 
-**Point of sale app** can also be used for receiving donations or even as a small e-shop, depending on the customizations applied.
+The **Point of sale app** can also be used to receive donations, tips or even be used as a small e-shop, depending on the options or customizations applied.
```

Comment:
hmm I wonder if you would say e-shop or just "e-commerce usage"

Reply:
```suggestion
The **Point of sale app** can also be used to receive donations, tips or even be used as a small e-commerce shop, depending on the options or customizations applied.
```

You're right, e-shop doesn't sound right.
How's the above ?